Python/Product/Uwp/Project/PythonUwpProject.cs

```diff
@@ -64,9 +67,33 @@ partial class PythonUwpProject :
         /// information from the project)
         /// </summary>
         protected override void InitializeForOuter(string fileName, string location, string name, uint flags, ref Guid guidProject, out bool cancel) {
+            var pythonProject = this.GetProject().GetPythonProject();
+            var msbuildProject = pythonProject.GetMSBuildProjectInstance();
+            msbuildProject.Build("CreatePythonUwpIoTPythonEnv", null);
+
+            var sitePackagesDir = Path.Combine(pythonProject.ProjectDirectory, PythonUwpConstants.InterpreterRelativePath, PythonUwpConstants.InterpreterLibPath, "site-packages");
```

Comment:
Why is `"site-packages"` the only one that isn't a constant?

Also be aware of potential long filename issues. They may break later operations, but we'd rather they didn't crash right here.